rake_lib/helper.rb

```diff
@@ -159,6 +159,8 @@ def update_docs is_master
       if is_master
         tags = %w[master]
       else
+        # Adding branches
+        tags = @appium_repo.documentation_publish_branches + tags
```

Comment:
branches are not tags. they are different concepts and handled separately in the code. I don't think mashing them together with tags is helpful.